client/src/components/ChatSideBanner/ChatSideBanner.tsx

```diff
@@ -1,21 +1,28 @@
-import { ChangeEvent, useState } from 'react';
+import { ChangeEvent, useState, useEffect } from 'react';
 import Box from '@material-ui/core/Box';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
 import useStyles from './useStyles';
 import { User } from '../../interface/User';
 import Search from '../Search/Search';
 import ConvoCard from '../ConvoCard/ConvoCard';
-
-interface Props {
-  handleDrawerToggle?: () => void;
-}
+import { Convo } from '../../interface/User';
+import { getAllConvos } from '../../helpers/APICalls/conversations';
 
 const ChatSideBanner = (): JSX.Element => {
   const [search, setSearch] = useState<string>('test');
+  const [convos, setConvos] = useState<Convo[]>([]);
   const [newChatUser, setNewChatUser] = useState<User | null>(null);
   const classes = useStyles();
 
+  useEffect(() => {
+    getAllConvos().then((data: Convo[]) => {
```

Comment:
The ChatSideBanner is rendered within the Dashboard component, each time the `friendId` is updated we are fetching all the conversations once again, this may will an impact on performance :)

Reply:
This is fixed.